docs/how-to-guides/polygon-id-state-replication.mdx

```diff
@@ -25,12 +25,14 @@ docker pull registry.gitlab.com/rarimo/polygonid/relayer-svc:cca1632497291b16a69
 573
 ```
 
-Or clone the  [repository](https://gitlab.com/rarimo/polygonid/relayer-svc) and build it from the source:
+Or clone the  [repository](https://github.com/rarimo/relayer-svc) and build it from the source:
```

Comment:
This is the correct repo: https://github.com/rarimo/identity-relayer-svc?tab=readme-ov-file